### PR TITLE
Adding to get OSTreeCommit from imagebuilder 

### PR DESCRIPTION
**Pull request by Saurabh-Thakre**
Tried explaining issues and steps I've done to test in the comments. Please do let me know if additional info is needed.

### CODE REVIEW

**Review by biancarosa — COMMENTED** [review]
You have:
`var i *models.Image`

but when you call `getImage(w, r)` you're assigning to the `image variable`.

Maybe you drop this `var i *models.Image` declaration and use image on your call on GetMetadata?

**Comment by Saurabh-Thakre** [comment]
> You have:
> `var i *models.Image`
> 
> but when you call `getImage(w, r)` you're assigning to the `image variable`.
> 
> Maybe you drop this `var i *models.Image` declaration and use image on your call on GetMetadata?

Yo done, and its working like charm

File: pkg/images/images.go [hunk 1]
```diff
@@ -182,6 +183,7 @@ func postProcessImage(id uint, headers map[string]string) {
 
 	// TODO: This is also where we need to get the metadata from image builder
 	// in a separate goroutine
+	go imagebuilder.Client.GetMetadata(i, headers)
```
Comment (biancarosa) [inline pkg/images/images.go:186]: Oh, we can actually move this to before we create the OSTree repo! After the for {} on line 166

**Review by biancarosa — APPROVED** [review]
LGTM!

**Pull request merged**